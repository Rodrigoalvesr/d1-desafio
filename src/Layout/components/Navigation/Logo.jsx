import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import logo from '@/assets/images/logotipo.png';

const useStyles = makeStyles({
  root: {
    display: 'flex',
    justifyContent: 'center',
    marginTop: 35,
    marginBottom: 35,
  },
});

function Logo() {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <a href="https://www.d1.cx/">
        <img alt="D1" src={logo} height={22} />
      </a>
    </div>
  );
}

export default Logo;
