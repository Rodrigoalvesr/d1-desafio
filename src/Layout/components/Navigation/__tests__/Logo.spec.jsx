import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../Logo';

describe('Logo', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(<Logo name="AnÃ¡lises" icon="chart" />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
