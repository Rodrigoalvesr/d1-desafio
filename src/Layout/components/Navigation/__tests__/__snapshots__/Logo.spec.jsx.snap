// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <a
    href="https://www.d1.cx/"
  >
    <img
      alt="D1"
      height={22}
      src="logotipo.png"
    />
  </a>
</div>
`;
